keyloop/api/v1/identity.py

```diff
@@ -70,13 +71,23 @@ def collection_post(self):
     @grip_view(schema=GetAndDeleteSchema, response_schema=get_response_schemas)
     def get(self):
         """ Return identity info + permissions """
-        return self.request.identity_provider.get(self.request.matchdict['id'])
+        return self.request.identity_provider.get(self.request.validated['path']['id'])
 
     @grip_view(schema=GetAndDeleteSchema, response_schema=collection_delete_put_response_schemas)
     def delete(self):
         """Remove the identity"""
 
-        self.request.identity_provider.delete(self.request.matchdict['id'])
+        try:
+            self.request.identity_provider.delete(self.request.validated['path']['id'])
+
+            return HTTPNoContent()
```

Comment:
Acho que fica mais fácil de entender se esse return for removido

Reply:
sem esse return não consigo passar que essa chamada tem que responder um 204 e não 200